staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -113,6 +113,15 @@ var (
 		},
 		[]string{"requestKind"},
 	)
+	// TLSHandshakeErrors is a number of requests dropped with 'TLS handshake error from' error
+	TLSHandshakeErrors = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_tls_handshake_errors_total",
+			Help:           "Number of requests dropped with 'TLS handshake error from' error",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{},
```

Comment:
Is it intentional to leave this blank? Why not just use Counter metric instead?

Reply:
I just copy-pasted it from other metrics and since there is no labels to pass, I left it empty. It should be changed indeed.